nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/blob/BlobProvider.java

```diff
@@ -209,4 +209,13 @@ default boolean performsExternalAccessControl(BlobInfo blobInfo) {
         return false;
     }
 
+    /**
+     * Returns the properties of the blob provider.
+     * 
+     * @since 10.2
+     */
+    default Map<String, String> getProperties() {
+        return Collections.emptyMap();
```

Comment:
Instead of a default method on the interface could you just also implement it for `AbstractCloudBinaryManager` (which is our only other base implementation of `BlobProvider` besides `AbstractBlobProvider`)? So that the `getProperties` API will really be usable on all our blob providers.